cmd/device-util/service.go

```diff
@@ -0,0 +1,144 @@
+//go:build windows
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"unsafe"
+
+	cli "github.com/urfave/cli/v2"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+var serviceCommand = &cli.Command{
+	Name:    "service",
+	Aliases: []string{"svc"},
+	Usage:   "manage Windows services",
+	Before:  verifyElevated,
+	Subcommands: []*cli.Command{
+		//todo (helsaawy): add stop and delete service commands
```

Comment:
I'm not a big fan of TODOs in code, but we should at least capitalize them so it's easier to find them via search.

Reply:
Removed it, since were trying to tie driver lifetime with the containers, and it doesnt seem necessary at all